boa3_test/tests/boatestcase.py

```diff
@@ -0,0 +1,544 @@
+
+        return cls.contract
+
+    @classmethod
+    async def compile_and_deploy(cls,
+                                 contract_path: str,
+                                 *,
+                                 output_name: str = None,
+                                 change_manifest_name: bool = False,
+                                 signing_account: account.Account = None
+                                 ) -> types.UInt160:
+
+        nef_abs_path, _ = cls.get_deploy_file_paths(contract_path,
+                                                    output_name=output_name,
+                                                    change_manifest_name=change_manifest_name,
+                                                    )
+
+        if not signing_account:
+            signing_account = cls.genesis
+
+        return await cls.deploy(nef_abs_path, signing_account)
+
+    def unwrap_inner_values(self, value: list | dict):
+        if isinstance(value, list):
+            for index, item in enumerate(value):
+                if not isinstance(item, noderpc.StackItem):
+                    continue
+
+                value[index] = self._unwrap_stack_item(item)
+
+        elif isinstance(value, dict):
+            aux = value.copy()
+            value.clear()
+            for key, item in aux.items():
+                dict_key = key if not isinstance(key, noderpc.StackItem) else self._unwrap_stack_item(key)
+                dict_item = item if not isinstance(item, noderpc.StackItem) else self._unwrap_stack_item(item)
+
+                value[dict_key] = dict_item
+
+    def _unwrap_stack_item(self, stack_item: noderpc.StackItem) -> Any:
+        result = None
+
+        if stack_item.type is noderpc.StackItemType.BYTE_STRING:
+            try:
+                result = stack_item.as_str()
+            except:
+                result = stack_item.as_bytes()
+
+        elif stack_item.type is noderpc.StackItemType.INTEGER:
+            result = stack_item.as_int()
+
+        elif stack_item.type is noderpc.StackItemType.BOOL:
+            result = stack_item.as_bool()
+
+        elif stack_item.type is noderpc.StackItemType.BUFFER:
+            try:
+                result = stack_item.as_uint160()
+            except ValueError:
+                try:
+                    result = stack_item.as_uint256()
+                except ValueError:
+                    try:
+                        result = stack_item.as_public_key()
+                    except ValueError:
+                        result = stack_item.value
+
+        elif stack_item.type is noderpc.StackItemType.ARRAY:
+            result = stack_item.as_list()
+            self.unwrap_inner_values(result)
+
+        elif stack_item.type is noderpc.StackItemType.MAP:
+            result = stack_item.as_dict()
+            self.unwrap_inner_values(result)
+
+        if result is None:
+            result = stack_item.value
+        return result
```

Comment:
Because the `unwrap` methods raise errors with StackItemType different from the `elif`s. It's from mamba code.
https://github.com/CityOfZion/neo-mamba/blob/6cb4351d4ca9af240bb90570c8ed28606e0ab4bd/neo3/api/noderpc.py#L331-L357